WooCommerce/Classes/ViewRelated/Dashboard/DashboardView.swift

```diff
@@ -121,8 +121,8 @@ struct DashboardView: View {
         }
         .sheet(isPresented: $viewModel.showingCustomization) {
             DashboardCustomizationView(viewModel: DashboardCustomizationViewModel(
-                allCards: viewModel.dashboardCards,
-                inactiveCards: viewModel.unavailableDashboardCards,
+                allCards: viewModel.dashboardCards.filter { $0.status != .hide },
+                inactiveCards: viewModel.dashboardCards.filter { $0.status == .unavailable },
```

Comment:
These are logic and should be moved to the view model for unit testing.

Reply:
Thanks! This is a great point and I moved display logic here and in the model, to the view model, in https://github.com/woocommerce/woocommerce-ios/pull/12626/commits/3fc44de92ea04425d36e625f6aab9e671c124d6a